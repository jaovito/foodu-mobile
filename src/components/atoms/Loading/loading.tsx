import {LoadingProps} from './loading.props';
import {ActivityIndicator, Container} from './loading.styles';

export const Loading = (props: LoadingProps) => {
  return (
    <Container>
      <ActivityIndicator size="large" {...props} />
    </Container>
  );
};
