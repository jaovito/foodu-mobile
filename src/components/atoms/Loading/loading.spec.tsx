import {render} from '@test-utils';
import {Loading} from './loading';

describe('<Loading /> tests', () => {
  it('should be defined', () => {
    const loading = render(<Loading />);

    expect(loading).toBeDefined();
  });

  it('should match with snapshot', () => {
    const loading = render(<Loading />).toJSON();

    expect(loading).toMatchSnapshot();
  });
});
