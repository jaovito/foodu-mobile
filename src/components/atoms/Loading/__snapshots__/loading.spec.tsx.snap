// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> tests should match with snapshot 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "rgba(0, 0, 0, 0.5)",
        "height": "100%",
        "justifyContent": "center",
        "position": "absolute",
        "width": "100%",
        "zIndex": 100,
      },
    ]
  }
>
  <ActivityIndicator
    size="large"
    style={
      [
        {},
      ]
    }
  />
</View>
`;
